feat: migrate from sync to async (#115)
webdriverio/appium-boilerplate: README.md

# appium-boilerplate

> **NOTE:**

> > This boilerplate is for Webdriver V7 where the tests are written in `async` mode and TypeScript.\

Reviewer: ```suggestion
> This boilerplate is for Webdriver V7 where the tests are written with `async`/`await` and TypeScript.\
```
